lib/ret/asset.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Ret.Asset do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Ret.{Asset}
+
+  @schema_prefix "ret0"
+  @primary_key {:asset_id, :id, autogenerate: true}
+
+  schema "assets" do
+    field(:asset_sid, :string)
+    field(:name, :string)
+    field(:type, :string)
```

Comment:
I think this should be refactored to use an enum (in elixir and postgres) -- see `enums.ex` and the relevant schema/migrations on how to set it up.